docs/guides/postgres/audit-logging/pgaudit.md

```
@@ -0,0 +1,553 @@
+```
+
+`source = configuration file` is the thing to look for. Anything you set that
+still reads `default` did not take effect.
+
+## Step 3 — Create the extension and the audit role
+
+Preloading the library is what makes auditing happen; `CREATE EXTENSION` adds
+the SQL-level objects and is still required for a supported installation.
+
+```bash
+$ kubectl exec -it -n bank bank-pg-0 -c postgres -- psql -U postgres
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)